policy/cnspec_policy.proto

```diff
@@ -415,10 +415,10 @@ message Reports {
 }
 
 message ReportCollection {
-  map<string, Asset> assets = 1;
```

Comment:
Is there really much use for the policy Asset type/message any more? I see it used in various places, but (from a quick glance) they seem like they could also just use the motor.Asset type?
Anyway, not really specific to the purpose of the PR. Just asking